scripts/install.js

```diff
@@ -131,13 +118,22 @@ async function downloadLibtensorflow(callback) {
   await ensureDir(depsPath);
 
   const request = https.get(targetUri, response => {
-    if (platform.endsWith('windows')) {
+    const bar = new ProgressBar('downloading [:bar] :rate/bps :percent :etas', {
```

Comment:
We can tackle this in a future PR - but my win machine still doesn't have a progress bar.